.github/workflows/ci.yml

```diff
@@ -653,6 +727,7 @@ jobs:
 
   upload-coverage:
     needs: [test-internals, test-api-and-no-engine, test-defaults, test-hdk, test-all-unidist, test-all, test-experimental, test-cloud]
+    if: github.event_name == 'push'
```

Comment:
Will we not see the coverage in the pull request itself? Annotations were helpful.

Reply:
I think they won't be useful in PRs when this one is merged, as the tests being run would be different from run to run, and coverage percentage would be jumping wildly...

Reply:
We can disable the percentage coverage check (in the PR), but still see rows that are not covered directly in the pull request.

Reply:
That said, I don't see any place where we even set this threshold now... was it removed earlier?

Reply:
I don't remember exactly, but it looks like that.